Dockerfile

```
@@ -161,7 +161,8 @@ RUN apk add --no-cache \
     # pre-commit deps
     git=~2 \
     # All hooks deps
-    bash=~5
+    bash=~5 \
+    alpine-sdk
```

Should the comment be added to this line to align with how other packages in this block are described?

---

I am not sure what you mean, do you mean should we add another comment to differentiate the hooks deps.
```suggestion
    # Third party hooks deps
    alpine-sdk
```

Like this ?

---

Yep, that's what I mean. Thanks.